app/views/unattended/provisioning_templates/registration/global_registration.erb

```diff
@@ -6,12 +6,14 @@ model: ProvisioningTemplate
 #!/bin/sh
 <%
     headers = ["-H 'Content-Type: application/json'", "-H 'Accept: application/json'", "-H 'Authorization: Bearer #{@auth_token}'"]
+    organization = @organization ||@user.my_organizations.first
```

Comment:
Could you please move this and the line below to the controller? The template should contain the least amount of logic possible. If you do that, we don't have to allow access to `my_organizations` and `my_locations` in the safe mode. If you want to keep them, we need dsl documentation for them too.

You can do it easily in [registration module](https://github.com/theforeman/foreman/blob/develop/app/controllers/concerns/foreman/controller/registration.rb#L27-L28)

Reply:
also, missing space after `||`

Reply:
and last but not least, I think the logic should be 
```
@organization || @user.default_organization || @user.my_organizations.first
```

(the same for location). In controller concern the code will look slightly differently.